src/components/scenes/WalletListScene.js

```diff
@@ -10,8 +10,8 @@ import { useSelector } from '../../types/reactRedux.js'
 import { getWalletListSlideTutorial, setUserTutorialList } from '../../util/tutorial.js'
 import { CrossFade } from '../common/CrossFade.js'
 import { SceneWrapper } from '../common/SceneWrapper.js'
+import { ButtonsModal } from '../modals/ButtonsModal.js'
 import { PasswordReminderModal } from '../modals/PasswordReminderModal.js'
-import { WalletListSlidingTutorialModal } from '../modals/WalletListSlidingTutorialModal.js'
```

Comment:
Since this file is not used anymore you should remove it

Reply:
Done